src/ducks/account/LoanDetailsPage.jsx

```diff
@@ -1,11 +1,260 @@
+import { Section as BaseSection } from 'components/Section'
+import cx from 'classnames'
+import styles from 'ducks/account/LoanDetailsPage.styl'
+
+const DATE_FORMAT = 'DD/MM/YY'
+
+const Row = translate()(props => {
+  const {
+    type,
+    title,
+    value: originalValue,
+    caption,
+    t, // eslint-disable-line no-unused-vars
+    f,
+    className,
+    ...rest
+  } = props
+
+  if (!originalValue) {
+    return null
+  }
+
+  let value
+
+  switch (type) {
+    case 'amount':
+      value = <Figure total={originalValue} symbol="€" />
+      break
+
+    case 'date':
+      value = f(originalValue, DATE_FORMAT)
+      break
+
+    case 'rate':
+      value = originalValue + '%'
+      break
+
+    case 'nb-payments':
+      value = 'x' + originalValue
+      break
+
+    default:
+      value = originalValue
+      break
+  }
+
+  const right = <span className={styles.LoanDetailsRow__value}>{value}</span>
+
+  return (
+    <div className={cx(styles.LoanDetailsRow, className)}>
+      <CompositeRow
+        className={cx(styles.LoanDetailsRow__inner)}
+        primaryText={title}
+        secondaryText={caption}
+        right={right}
+        {...rest}
+      />
+    </div>
+  )
+})
+
+const Section = props => {
+  const { title, children } = props
+
+  return <BaseSection title={title}>{children}</BaseSection>
+}
+
+const hasInfos = (account, paths) => {
+  for (const path of paths) {
+    if (get(account, path)) {
+      return true
+    }
+  }
+
+  return false
+}
+
+const KeyInfosSection = translate()(props => {
+  const { account, t } = props
+
+  const shouldRender = hasInfos(account, [
+    'loan.usedAmount',
+    'balance',
+    'loan.rate'
+  ])
+
+  if (!shouldRender) {
+    return null
+  }
+
+  return (
+    <Section title={t('Informations clés')}>
+      <Row
+        type="amount"
+        title={t('Montant emprunté')}
+        value={get(account, 'loan.usedAmount')}
+      />
+      <Row
+        type="amount"
+        title={t('Capital restant dû')}
+        value={-get(account, 'balance')}
+      />
+      <Row
+        type="rate"
+        title={t("Taux d'intérêt")}
+        value={get(account, 'loan.rate')}
+      />
+    </Section>
+  )
+})
+
+const PaymentsSection = translate()(props => {
+  const { account, t, f } = props
+
+  const shouldRender = hasInfos(account, [
+    'loan.lastPaymentDate',
+    'loan.nextPaymentDate'
+  ])
+
+  if (!shouldRender) {
+    return null
+  }
+
+  return (
+    <Section title={t('Paiements')}>
+      <Row
+        type="amount"
+        title={t('Dernier paiement effecté')}
+        value={get(account, 'loan.lastPaymentAmount')}
+        caption={'le ' + f(get(account, 'loan.lastPaymentDate'), DATE_FORMAT)}
+      />
+      <Row
+        type="amount"
+        title={t('Prochain paiement prévu')}
+        value={get(account, 'loan.nextPaymentAmount')}
+        caption={
+          t('le ') + f(get(account, 'loan.nextPaymentDate'), DATE_FORMAT)
+        }
```

Comment:
Careful you seem have untranslated chains here. In my latest works, I used {/* i18n /*} comments when I was building the page, not to forget to remove them afterwards.